benchmark/scripts/benchmark_cpo_loss.py

```diff
@@ -62,15 +40,21 @@ def forward(self, x, y):
 def bench_memory_fused_linear_cpo_loss(
     input: SingleBenchmarkRunInput,
 ) -> SingleBenchmarkRunOutput:
+    from test.chunked_loss.test_cpo_loss import TorchLMHeadCPO
```

Comment:
Curious why not also use LigerLMHeadCPO from tests if we're using TorchLMHeadCPO here

Reply:
Ah certainly! Thanks. Updated~